### ISSUE TITLE
Update User Filters on older data sources

### ISSUE THREAD

**Issue by mcoles** [tableau/community-tableau-server-insights#16]
We created some new features for data sources in later 02 version sets that were never added to the old 01 version set. Add new permissions strings to it and edit the User Filter calc appropriately.

**Comment by mcoles** [comment]
TS Events uses an old-school method, so both 02 and 01 version sets need to be updated. I'm a bit concerned that performance will tank, however, even if the field goes unused, since we'll have to join a bunch of CustomSQL into it...I suppose I'll add them in, then test live query and extract speed differences.

**Comment by mcoles** [comment]
Updating this for posterity: This took a long time for TS Events, because it required joining a crapload more tables to an already spiderwebby connection structure. I was concerned what this might do to performance, so at first I tried converting it to a UNION structure rather than doing a million joins. This actually TANKED performance. So I resigned myself to joining all the additional project, user, and system_user tables to all four of the branches in the connection, and did testing against the prior version.

Live connections are very slow to render, but not worse than the prior version:

![image](https://user-images.githubusercontent.com/14928335/71131137-b7cb5580-21a8-11ea-99be-5ecc56338847.png)

Extract refreshes also seemed not to suffer much:

![image](https://user-images.githubusercontent.com/14928335/71131237-f8c36a00-21a8-11ea-90ce-a3f22ba36f80.png)

So, yay
